### PR TITLE
feat(gateway): adds gateway commands and actions

### LINKED ISSUE #3: (api.channelGateway)[b]: Implement action, validator, and schema

### PR DESCRIPTION
**Pull request by CodeJonesW**
fixes #3

### CODE REVIEW

File: lib/enums/channel.ts [hunk 1]
```diff
@@ -0,0 +1,107 @@
+export const CHANNEL_CHANNEL_COMMANDS = {
```
Comment (scott-wyatt) [inline lib/enums/channel.ts]: We should probably just call this `CHANNEL_COMMANDS`

File: lib/enums/channel.ts [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+    'ADD_RULES_SHIPPING': 'add.channel.:channel_uuid.rules_shipping',
+    'UPDATE_RULES_SHIPPING': 'update.channel.:channel_uuid.rules_shipping',
+    'REMOVE_RULES_SHIPPING': 'remove.channel.:channel_uuid.rules_shipping',
+
+    'ADD_RULES_TAXES': 'add.channel.:channel_uuid.rules_taxes',
+    'UPDATE_RULES_TAXES': 'update.channel.:channel_uuid.rules_taxes',
+    'REMOVE_RULES_TAXES': 'remove.channel.:channel_uuid.rules_taxes'
+}
+export const CHANNEL_CHANNEL_EVENTS = {
```
Comment (scott-wyatt) [inline lib/enums/channel.ts]: `CHANNEL_EVENTS`

File: lib/enums/channel.ts [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    'RULES_TAXES_ADDED': 'channel.:channel_uuid.rules_taxes.added',
+    'RULES_TAXES_UPDATED': 'channel.:channel_uuid.rules_taxes.updated',
+    'RULES_TAXES_REMOVED': 'channel.:channel_uuid.rules_taxes.removed',
+
+    'RULES_SHIPPING_ADDED': 'channel.:channel_uuid.rules_shipping.added',
+    'RULES_SHIPPING_UPDATED': 'channel.:channel_uuid.rules_shipping.updated',
+    'RULES_SHIPPING_REMOVED': 'channel.:channel_uuid.rules_shipping.removed'
+
+}
+export const CHANNEL_CHANNEL_ACTIONS = {
```
Comment (scott-wyatt) [inline lib/enums/channel.ts]: `CHANNEL_ACTIONS`

**Comment by scott-wyatt** [comment]
Looks good.  Just needs tests in place, it's fine to use `it.skip` for all them for right now, but we at least want the test there even it's not running :-)

File: test/integration/endpoints/channelGateway/gateway.test.js [hunk 4]
```diff
@@ -116,5 +141,157 @@ describe('# RiSE Channel Gateway API', () => {
           done(err)
         })
     })
+
+    it.skip('### Should Publish a Gateway', (done) => {
+
+      rise.channelGateway.list({
```
Comment (scott-wyatt) [inline test/integration/endpoints/channelGateway/gateway.test.js]: Could we convert these from .list to the actual method?   :-)

Comment (CodeJonesW) [reply test/integration/endpoints/channelGateway/gateway.test.js]: yep yep

**Pull request merged**